### PR TITLE
add github action to deploy storybook

### PR DESCRIPTION
**Pull request by CITIZENDOT**
## Summary
Whenever there's a change in `ui` package, this new github action gets triggered and pushes the static storybook files to [storybook-docs](https://github.com/ir-engine/storybook-docs) repo. Then it gets published through github pages.

## References
closes #_insert number here_

### CODE REVIEW

File: .github/workflows/update-storybook.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+          git config --global user.name "github-actions[bot]"
+          git config --global user.email "github-actions[bot]@users.noreply.github.com"
+
+          git clone https://x-access-token:${{ secrets.STORYBOOK_DOCS_PAT }}@github.com/your-username/storybook-docs.git
+          cp -R packages/ui/storybook-static/* storybook-docs/
+          
+          cd storybook-docs
+          touch .nojekyll
+          git add .
+          git commit -m "Update Storybook documentation"
```
Comment (aditya-mitra) [inline .github/workflows/update-storybook.yml:44]: `commit --amend`

git history will not be necessary for generated files

**Review by aditya-mitra — CHANGES_REQUESTED** [review]

File: .github/workflows/update-storybook.yml (hunk 1 above)
Comment (CITIZENDOT) [reply .github/workflows/update-storybook.yml:44]: updated

**Review by aditya-mitra — APPROVED** [review]
Awesome 🚀

File: packages/ui/postcss.config.js [hunk 2]
```diff
@@ -25,7 +25,7 @@ Infinite Reality Engine. All Rights Reserved.
 */
 
 
-module.exports = {
+export default {
```
Comment (HexaField) [inline packages/ui/postcss.config.js]: need to revert this as we reverted using ESM

**Review by HexaField — CHANGES_REQUESTED** [review]
Can this action be in the storybook repo instead? It can listen for changes to this repository with event webhooks. What are the tradeoffs of having it in this repo vs the other repo?

File: packages/ui/postcss.config.js (hunk 2 above)
Comment (CITIZENDOT) [reply packages/ui/postcss.config.js]: reverted

**Comment by CITIZENDOT** [comment]
> Can this action be in the storybook repo instead?

Yes, that is possible. Instead of creating/using secret token, webhook needs to be created.

**Comment by HexaField** [comment]
Let's do that then.

**Pull request closed without merging**